completions/man

```diff
@@ -51,7 +51,8 @@ _man()
         return
     fi
 
-    local manpath=$( manpath 2>/dev/null || command man -w 2>/dev/null )
+    local manpath=$( manpath 2>/dev/null || command man -w 2>/dev/null || command man --path 2>/dev/null )
```

Comment:
Are there systems have support for `--path` but not `-w`? See commit 492773098004700d670b4e1c12eebac506471d90